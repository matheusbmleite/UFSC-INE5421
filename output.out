{
    "transitions": {
        "['q', 'r', 'p']": {
            "a": [
                "q",
                "p"
            ],
            "c": [
                "q",
                "r",
                "p"
            ],
            "b": [
                "q",
                "r"
            ]
        },
        "['q', 'r']": {
            "a": [
                "q",
                "p"
            ],
            "c": [
                "q",
                "p"
            ],
            "b": [
                "r"
            ]
        },
        "['r']": {
            "Îµ": [],
            "a": [],
            "c": [],
            "b": []
        },
        "['q', 'p']": {
            "a": [
                "q",
                "p"
            ],
            "c": [
                "q",
                "r",
                "p"
            ],
            "b": [
                "q",
                "r"
            ]
        }
    },
    "alphabet": [
        "a",
        "b",
        "c"
    ],
    "type": "automaton",
    "states": [
        [
            "q",
            "r"
        ],
        [
            "r"
        ],
        [
            "q",
            "r",
            "p"
        ],
        [
            "q",
            "p"
        ]
    ]
}