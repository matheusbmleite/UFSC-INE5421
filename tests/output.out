{
    "alphabet": [
        "a",
        "c",
        "b"
    ],
    "type": "automaton",
    "transitions": {
        "['p', 'q']": {
            "a": [
                "p",
                "q"
            ],
            "c": [
                "p",
                "r",
                "q"
            ],
            "b": [
                "q",
                "r"
            ]
        },
        "['q', 'r']": {
            "a": [
                "p",
                "q"
            ],
            "c": [
                "p",
                "q"
            ],
            "b": [
                "r"
            ]
        },
        "['p', 'q', 'r']": {
            "a": [
                "p",
                "q"
            ],
            "c": [
                "p",
                "r",
                "q"
            ],
            "b": [
                "q",
                "r"
            ]
        },
        "['r']": {
            "a": [],
            "Îµ": [],
            "c": [],
            "b": []
        }
    },
    "init_state": [
        "p",
        "q"
    ],
    "states": [
        [
            "r"
        ],
        [
            "p",
            "q"
        ],
        [
            "p",
            "q",
            "r"
        ],
        [
            "q",
            "r"
        ]
    ],
    "final_states": [
        [
            "r"
        ],
        [
            "p",
            "q",
            "r"
        ],
        [
            "q",
            "r"
        ]
    ]
}